src/web-push-lib.js

```diff
@@ -18,6 +18,11 @@ function WebPushLib() {
 
 }
 
+WebPushLib.supportedContentEncodings = {
```

Comment:
I wish this was available in vapid-helper.js too, so that we wouldn't need to hardcode the strings.

Reply:
Should I just require the web push lib in vapid-helper.js?

Reply:
I don't know if that would work, as web-push-lib is importing vapid-helper, so this would create an infinite loop.

Maybe we can put this variable into a separate web-push-constants.js module (even though it would be a really small module 😄 )